Collector/ProfileFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Http\HttplugBundle\Collector;
+
+/**
+ * @author Fabien Bourigault <bourigaultfabien@gmail.com>
+ *
+ * @internal
+ */
+class ProfileFactory
```

Comment:
ah, i see. yeah lets make all 4 classes final then, to avoid people extending them. both profile / stack and the factories.